recirq/qcqmc/fqe_conversion.py

```diff
@@ -0,0 +1,246 @@
+    """Create the qubit representation of Full CI according to the parameters passed.
+
+    Args:
+        norb: number of spatial orbitals
+        nele: number of electrons
+        m_s: spin projection onto sz
+        fermion_ind_to_qubit_ind: a dictionary that describes the assignment of
+            the fermionic modes to the qubit modes.
+
+    Returns:
+        ops
+    """
+    fermion_op = fqe_of.fci_fermion_operator_representation(norb, nele, m_s)
+
+    # We might need to reorder the terms.
+    if fermion_ind_to_qubit_ind is not None:
+        num_modes = len(fermion_ind_to_qubit_ind.items())
+
+        def mapper(idx, num_modes):
+            return fermion_ind_to_qubit_ind[idx]
+
+        fermion_op = of.reorder(fermion_op, mapper, num_modes=num_modes)
+
+    return of.jordan_wigner(fermion_op)
+
+
+def get_reorder_func(
+    *,
+    mode_qubit_map: Mapping[fm.FermionicMode, cirq.Qid],
+    ordered_qubits: Sequence[cirq.Qid],
+) -> Callable[[int, int], int]:
+    """This is a helper function that allows us to reorder fermionic modes.
+
+    Under the Jordan-Wigner transform, each fermionic mode is assigned to a
+    qubit. If we are provided an openfermion FermionOperator with the modes
+    assigned to qubits as described by mode_qubit_map this function gives us a
+    reorder_func that we can use to reorder the modes (with
+    openfermion.reorder(...)) so that they match the order of the qubits in
+    ordered_qubits. This is necessary to make a correspondence between
+    fermionic operators / wavefunctions and their qubit counterparts.
+
+    Args:
+        mode_qubit_map: A dict that shows how each FermionicMode is mapped to a qubit.
+        ordered_qubits: An ordered sequence of qubits.
+    """
+    qubits = list(mode_qubit_map.values())
+    assert len(qubits) == len(ordered_qubits)
+
+    # We sort the key: value pairs by the order of the values (qubits) in
+    # ordered_qubits.
+    sorted_mapping = list(mode_qubit_map.items())
+    sorted_mapping.sort(key=lambda x: ordered_qubits.index(x[1]))
+
+    remapping_map = {}
+    for i, (mode, _) in enumerate(sorted_mapping):
+        openfermion_index = 2 * mode.orb_ind + (0 if mode.spin == "a" else 1)
+        remapping_map[openfermion_index] = i
+
+    def remapper(index: int, _: int) -> int:
+        """A function that maps from the old index to the new one.
+
+        The _ argument is because it's expected by openfermion.reorder"""
+        return remapping_map[index]
+
+    return remapper
+
+
+def convert_qubit_wfn_to_fqe_syntax(
+    ops: of.QubitOperator, fermion_ind_to_qubit_ind: Optional[Dict[int, int]] = None
+) -> of.FermionOperator:
+    """Converts qubit wavefunction to FQE syntax.
+
+    This takes a qubit wavefunction in the form of a string of qubit operators
+    and returns a string of FermionOperators with the proper formatting for easy
+    digestion by FQE.
+
+    Args:
+        ops (QubitOperator) - a string of qubit operators
```

Comment:
nit:  Reformat these args to use colons, etc.

Reply:
Done.